app/models/schema.rb

```diff
@@ -54,6 +54,10 @@ def fields
     field_names.map { |field_name| Field.new(field_name, self) }
   end
 
+  def field(name)
+    fields.find { |f| f.name == name }
```

Comment:
would it be helpful to raise an error here if the expected field isn't found? 

e.g. with something like:

```rb
fields.find(proc { raise "field '#{name}' not found" } ) { |f| f.name == name }
```